libfuse/tlf.go

```diff
@@ -131,6 +131,17 @@ func (tlf *TLF) loadDirAllowNonexistent(ctx context.Context) (
 	return tlf.loadDirHelper(ctx, libkbfs.ReadMode, true)
 }
 
+// Access implements the fs.NodeAccesser interface for *TLF.
+func (tlf *TLF) Access(ctx context.Context, r *fuse.AccessRequest) error {
```

Comment:
The logic here should be the same as `Dir.Access()`, in that it should check write permission using the handle.  Unfortunately, `tlf.dir` might be `nil`, so you can either duplicate `Dir.Access()`, or move it up to the `Folder` level so both can share it.